proposals/proposals.py

```diff
@@ -1,5 +1,6 @@
 """
-Proposals module handles the logic associated with proposals. These functions are invoked when the gov commands
+proposals/proposals.py
```

Comment:
prop not important to include the file path. Unless there's something about this format that I dont know?

Reply:
> prop not important to include the file path. Unless there's something about this format that I dont know?

Nothing important about it, i'll remove the file path mentions.